src/storage/src/monitor/monitored_store.rs

```diff
@@ -359,9 +361,10 @@ impl MonitoredStateStore<HummockStorage> {
 }
 
 /// A state store iterator wrapper for monitoring metrics.
-pub struct MonitoredStateStoreIter<S> {
+pub struct MonitoredStateStoreIter<'s, S: 's> {
```

Comment:
Do we really need this `'s`? 👀

Reply:
@wenym1 PTAL

Reply:
`'a` is the lifetime of the future, and `'s` is the lifetime of the returned stream. They are not necessarily the same. For iterator returned from local state store, the `'s` is the lifetime of the local state store, while for iterator returned from global state store, its lifetime is static.